types/trigger.d.ts

```diff
@@ -200,6 +200,26 @@ export type TriggerSet<Data extends RaidbossData = RaidbossData> =
 // Less strict type for user triggers + built-in triggers, including deprecated fields.
 export type LooseTimelineTrigger = Partial<TimelineTrigger<RaidbossData>>;
 
+/**
+ *  @deprecated
+ */
+export type LegacyTrigger = {
```

Comment:
I think this could be combined into `LooseTrigger`.  `LooseTrigger` is the "user trigger, that could be missing fields, and could have old deprecated fields in it" type.  It's the same reason that `LooseTriggerSet` has `zoneRegex` in it.